charts/newrelic-infra-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}
+  labels:
+    {{- include "newrelic-infra-operator.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      {{- include "newrelic-infra-operator.appLabel" . | nindent 6 }}
+  template:
+    metadata:
+      {{- if .Values.podAnnotations }}
+      annotations:
```

Comment:
I'd add https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments to automatically restart operator on configuration change.

Reply:
I agree, it is quite annoying

Reply:
@invidian added